enzyme/Enzyme/TypeAnalysis/TypeAnalysis.cpp

```diff
@@ -4051,7 +4057,9 @@ TypeResults TypeAnalysis::analyzeFunction(const FnTypeInfo &fn) {
   }
 
   analysis.prepareArgs();
+  analysis.considerRustDebugInfo();
   analysis.considerTBAA();
+  auto fnName = fn.Function->getName();
```

Comment:
Is fnName needed here?

Reply:
No, it's a mistake. I'll delete it later.